### ISSUE TITLE
Merge conflicts interface disappears when going to next file

### ISSUE THREAD

**Issue by adambuczynski** [smashwilson/merge-conflicts#208]
It refuses to show when going to the next file. Have to quit the merge conflicts panel, and then find merge conflicts again with the relevant file open before it shows the interface again.

This is on Atom 1.6.0-beta5, OS X.

**Comment by jdinartejesus** [comment]
I'm also having the same issue as @adambuczynski and i'm using Atom 1.5.4 and OS X.

**Comment by trainerbill** [comment]
+1 same issue on 1.5.3

**Comment by smashwilson** [comment]
Do you have editor-config installed as well? See #196 for details if you do.

Are you on Windows? If so, #227 may be what you're encountering instead.

**Comment by adambuczynski** [comment]
Yes, I do have editor-config installed. I will have a look soon and see if it's still happening with the latest versions of both packages.

**Comment by adambuczynski** [comment]
I don't know, I can't manage to test this. I've tried to create merge conflicts in two files, but then it only picked up one of the files. I'll wait till I have an actual real merge conflict in multiple files happening and will then retest again.
